examples/test_cpp/main.cc

```diff
@@ -19,12 +19,12 @@ po::variables_map process_cmd_line(const int argc, char** argv) {
 	  ("features,x", po::value<size_t>()->default_value(10), "Features count")
     ("actions,a", po::value<size_t>()->default_value(2), "Number of actions")
     ("experiment_name,e", po::value<std::string>()->required(), "experiment name")
-    ("perf,p", "if it is perf test (otherwise - validity)")
     ("float_outcome,f", "if outcome is float (otherwise - json)")
     ("sleep,s", po::value<size_t>()->default_value(0), "Milliseconds to sleep between loop iterations")
     ("duration,d", po::value<size_t>(), "Duration of experiment (in ms). Alternative to n")
     ("instances,i", po::value<size_t>()->default_value(1), "Number of test loop instances")
     ("reward_period,r", po::value<size_t>()->default_value(0), "Ratio period (0 - no reward, otherwise - every $reward_period interaction is receiving reward)")
+    ("slots,q", po::value<size_t>()->default_value(0), "Number of slots (ccb simulation is running if > 0)")
```

Comment:
CB is kind of like 1 slot CCB, maybe change the wording here?

Reply:
we still support CCB stack (request_decision/ccb reduction etc) with single slot and this is separate code path from CB - so we should be able to test it.

Reply:
yeah good point